### PR TITLE
Overhaul paper

### PR DESCRIPTION
**Pull request by apragsdale**
This is a WIP. Lots of cleanup and filling in to do.

Essentially, this streamlines the main text of the paper and adds details to the Appendix. This also adds a new "showcase" illustration as figure 1.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -163,65 +167,284 @@ \section*{Introduction}
...
+\subsection*{Population genetics model}
+
+Demographic models consist of one or more populations (or ``demes'') defined
+by their size histories and the time intervals of their existence.
+Individuals can move between populations based on their ancestor-descendant
+relationships (i.e., the topology) or by continuous or discrete migration events.
+The deomgraphic model itself omits information about genome biology, such as
+genome size and architecture, ploidy, and mutation and recombination rates.
+It also omits any information about selection operating within populations.
+Within populations, allele-frequency dynamics are implicitly assumed to
```
Comment (jeromekelleher) [inline paper.tex]: Why "implicitly" here - aren't we explicitly just saying it now? I would just delete the word.

File: paper.tex [hunk 2]
```diff
@@ -163,65 +167,284 @@ \section*{Introduction}
...
+follow the Wright-Fisher model, so that generations are discrete and
+non-overlapping and population sizes are independent of mean fitness
+(i.e., ``soft'' selection~\citep{christiansen1975hard}).
+While these restrictions preclude some population-genetic scenarios of
+interest, such as explicit spatial structure or
```
Comment (jeromekelleher) [inline paper.tex]: This is explicit - maybe "continuous spatial structure" with a few cites? We can drop a note and revisit.

File: paper.tex [hunk 3]
```diff
@@ -163,65 +167,284 @@ \section*{Introduction}
...
+\aprcomment{stochastic carrying-capacity models, or something like that},
+many current forwards- and backwards-in-time simulation software
+make these same assumptions, so that Demes is relevant to a range of
+widely-used simulators.
```
Comment (jeromekelleher) [inline paper.tex:266]: I guess these are also the basic assumptions also shared by inference methods, which is important. Mabye another sentence here making this explicit, and listing a few different types of inference methods and giving citations? This is critical - we want to establish that Demes is the "middleware" between the inference methods and the simulators, and captures the basic assumptions made by them all.

File: paper.tex [hunk 4]
```diff
@@ -163,65 +167,284 @@ \section*{Introduction}
...
+In Appendix~\ref{sec:appendix-pop-gen-model}, we expand on these concepts
+and provide more details on the population genetics models supported by Demes.
+
+\subsection*{High-level specification}
+
+The high-level description of Demes models is focused on efficient human
+understanding and avoiding errors.
+We have adopted the widely used YAML format~\citep{ben2009yaml} as the
+primary interface for writing and interchangin demographic models
```
Comment (jeromekelleher) [inline paper.tex]:
```suggestion
primary interface for writing and interchanging demographic models
```

**Review by jeromekelleher — APPROVED** [review]
Fabulous!

A few drive by comments, but merge-n-iterate is my vote here.

File: paper.tex (hunk 3 above)
Comment (apragsdale) [reply paper.tex:266]: I added your comment to the tex file as a comment, to come back to.

File: paper.tex (hunk 2 above)
Comment (apragsdale) [reply paper.tex]: Done

File: paper.tex (hunk 1 above)
Comment (apragsdale) [reply paper.tex]: Done

**Comment by apragsdale** [comment]
I'll merge this now. Plan to get back to some editing later this week.

**Pull request merged**